lib/inspec/base_cli.rb

```diff
@@ -6,6 +6,20 @@
 require 'inspec/log'
 require 'inspec/profile_vendor'
 
+# Allow end of options during array type parsing
+# https://github.com/erikhuda/thor/issues/631
+class Thor::Arguments
```

Comment:
Code, docs and examples are all great - though I have the usual reservations about monkeypatching, I think we have no other practical choice here.  Thanks @jquick!